### DISCUSSION TITLE
v2.77.1

### DISCUSSION THREAD

**Discussion by nunocoracao** [nunocoracao/blowfish#1719] [Announcements]
## What's Changed
* 🔖 v2.77.1 by @nunocoracao in https://github.com/nunocoracao/blowfish/pull/1718

**Full Changelog**: https://github.com/nunocoracao/blowfish/compare/v2.77.0...v2.77.1

<hr /><em>This discussion was created from the release <a href='https://github.com/nunocoracao/blowfish/releases/tag/v2.77.1'>v2.77.1</a>.</em>

**Comment by john1-design** [comment]
Awesome work Nuno !! I sent you a message on LinkedIn my Name is John De SIlva. Once i get the backgroud working i will buy you 100 coffees

**Comment by designproworks** [comment]
Thank again can you add me to the authors list.
